client/www/_emails/markdown/may2026.md

```
@@ -0,0 +1,89 @@
+
+**Starter templates for Vue and Svelte**
+
+![](https://www.instantdb.com/img/emails/may2026/s_AA7FE922FB40128B92DB685D6EFC801C26C6290C432EAB84E4F6E7E3E3E3101F_1778716142582_starter-templates.png)
+
+If you’ve been building with Vue or Svelte and wished you could use Instant, now you can.
+
+Realtime queries, auth, presence, cursors, storage, streams: it’s all there. The Svelte SDK is built on Svelte 5 runes and the Vue SDK is built on Vue 3’s Composition API, so everything feels native.
+
+You can try them out by running:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)